specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -572,515 +567,531 @@
-          "type": "array",
-          "description": "List subscription impacted by the service health event.",
-          "items": {
+        }
+      },
+      "impactedResourceListResult": {
+        "properties": {
+          "value": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/impactedResourceStatus"
+            },
+            "description": "The list of impactedResourceStatus."
+          },
+          "nextLink": {
             "type": "string",
-            "description": "Subscription impacted by the service health event."
+            "description": "The URI to fetch the next page of impactedResourceStatus."
           }
         },
-        "lastUpdateTime": {
-          "type": "string",
-          "description": "It provides the Timestamp for when the last update for the service health event.",
-          "format": "date-time"
-        },
-        "updates": {
-          "type": "array",
-          "description": "List of updates for given service health event.",
-          "items": {
-            "$ref": "#/definitions/update"
+        "required": [
+          "value"
+        ],
+        "description": "The List impactedResourceStatus operation response."
+      },
+      "impactedResourceStatus": {
+        "type": "object",
+        "description": "impactedResource with health status",
+        "properties": {
+          "properties": {
+            "x-ms-client-flatten": true,
+            "type": "object",
+            "description": "Properties of impacted resource status.",
+            "properties": {
+              "availabilityState": {
+                "type": "string",
+                "description": "Impacted resource status of the resource.",
+                "enum": [
+                  "Available",
+                  "Unavailable",
+                  "Degraded",
+                  "Unknown"
+                ],
+                "x-ms-enum": {
+                  "name": "AvailabilityStateValues",
+                  "modelAsString": true
+                }
+              },
+              "title": {
+                "type": "string",
+                "description": "Title description of the impacted resource status."
+              },
+              "summary": {
+                "type": "string",
+                "description": "Summary description of the impacted resource status."
+              },
+              "reasonType": {
+                "type": "string",
+                "description": "When the resource's availabilityState is Unavailable, it describes where the health impacting event was originated.",
+                "enum": [
+                  "Unplanned",
+                  "Planned",
+                  "UserInitiated"
+                ],
+                "x-ms-enum": {
+                  "name": "ReasonTypeValues",
+                  "modelAsString": true
+                }
+              },
+              "occuredTime": {
```

Comment:
It exists already, but if you're changing the contract, "occured" should be "occurred"

Reply:
This path is used by a partner. The areas I am changing for the contract are not in use by any one.